src/Gateway.php

```diff
@@ -207,7 +212,16 @@ public function get_supported_payment_methods() {
 	 * @return string|null
 	 */
 	public function get_webhook_url( Payment $payment ) {
-		$url = \rest_url( Integration::REST_ROUTE_NAMESPACE . '/webhook/' . (string) $payment->get_id() );
+		$path = [
+			Integration::REST_ROUTE_NAMESPACE,
+			'webhook',
+			$this->should_create_order_for_payment( $payment ) ? 'order' : null,
+			$payment->get_id(),
+		];
+
+		$path = \array_filter( $path );
+
+		$url = \rest_url( \implode( '/', $path ) );
```

Comment:
Should we immediately make a clearer distinction?

- `wp-json/pronamic-pay/mollie/v1/payments/webhook`
- `wp-json/pronamic-pay/mollie/v1/orders/webhook`

or:

- `wp-json/pronamic-pay/mollie/v1/webhook/payments/`
- `wp-json/pronamic-pay/mollie/v1/webhook/orders/`

https://docs.mollie.com/overview/webhooks

Reply:
Discussed yesterday that the following notation seems best:

- `/wp-json/pronamic-pay/mollie/v1/payments/webhook/<payment_id>`
- `/wp-json/pronamic-pay/mollie/v1/orders/webhook/<payment_id>`

Reply:
Work in progress in https://github.com/pronamic/wp-pronamic-pay-mollie/commit/6d078aa94e3640d94673dc15174ecda43096fedd, @rvdsteege is working on the implementation of the order webhook.